configure.py

```diff
@@ -1117,6 +1119,25 @@ def query_seastar_flags(seastar_pc_file, link_static_cxx=False):
     modes[mode]['seastar_cflags'] = seastar_cflags
     modes[mode]['seastar_libs'] = seastar_libs
 
+MODE_TO_CMAKE_BUILD_TYPE = {'release' : 'RelWithDebInfo', 'debug' : 'Debug', 'dev' : 'Dev', 'sanitize' : 'Sanitize' }
+
+def configure_zstd(build_dir, mode):
+    zstd_build_dir = os.path.join(build_dir, mode, 'zstd')
+
+    zstd_cmake_args = [
+        '-DCMAKE_BUILD_TYPE={}'.format(MODE_TO_CMAKE_BUILD_TYPE[mode]),
+        '-DCMAKE_C_COMPILER={}'.format(args.cc),
+        '-DCMAKE_CXX_COMPILER={}'.format(args.cxx),
+        '-DCMAKE_C_FLAGS={}'.format(zstd_cflags),
+        '-DZSTD_BUILD_PROGRAMS=OFF'
+    ]
+
+    zstd_cmd = ['cmake', '-G', 'Ninja', os.path.relpath('zstd/build/cmake', zstd_build_dir)] + zstd_cmake_args
+
+    print(zstd_cmd)
```

Comment:
Debug leftover?

Reply:
Actually, no: I followed Seastar's behavior, which prints the executed cmake command (see configure.py in Seastar repo, end of configure_mode function).

Reply:
Fair enough. Just wanted to check. :)